parsers/lib/IN.py

```diff
@@ -16,15 +16,3 @@ def read_text_from_span_id(html, span_id):
 def read_value_from_span_id(html, span_id):
     html_span = read_text_from_span_id(html, span_id)
     return float(html_span)
-
-
-def read_datetime_with_only_time(time_string, time_format, now=utcnow()):
-    utc = now.floor("hour")
-    india_now = utc.to(TZ)
-    time = get(time_string, time_format)
-    india_date_time = india_now.replace(
-        hour=time.hour, minute=time.minute, second=time.second
-    )
-    if india_date_time > india_now:
-        india_date_time.shift(days=-1)
-    return india_date_time
```

Comment:
This was only used once, so I removed it and its test, and made an 2-liner solution to replace it.